packages/chain-adapters/src/evm/EvmBaseAdapter.ts

```diff
@@ -179,6 +182,11 @@ export abstract class EvmBaseAdapter<T extends EvmChainId> implements IChainAdap
         symbol: 'MATIC',
         explorer: 'https://polygonscan.com/',
       },
+      [KnownChainIds.GnosisMainnet]: {
+        name: 'Gnosis',
```

Comment:
```suggestion
        name: 'xDAI',
```

Reply:
Re-opening for visibility. `Gnosis` is correct as it refers to the chain name. However  this mapping is dumb and other values are wrong. I guess the intent that this is a chains config mapping vs. an assets list is not clear, will do a separe smallish PR to improve this.

Reply:
Disregard me, the naming is quite confusing here - should be the asset name and symbol, not the chain name and asset symbol. Committed suggestion